apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/Claimable.tsx

```diff
@@ -0,0 +1,646 @@
+"use client";
+import { DatePickerWithRange } from "@/components/ui/DatePickerWithRange";
+import { Spinner } from "@/components/ui/Spinner/Spinner";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { Alert, AlertTitle } from "@/components/ui/alert";
+import { Button } from "@/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { Separator } from "@/components/ui/separator";
+import { Skeleton } from "@/components/ui/skeleton";
+import { ToolTipLabel } from "@/components/ui/tooltip";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useMutation } from "@tanstack/react-query";
+import { CircleAlertIcon, PlusIcon, Trash2Icon } from "lucide-react";
+import { useCallback } from "react";
+import { useFieldArray, useForm } from "react-hook-form";
+import { toast } from "sonner";
+import { type PreparedTransaction, sendAndConfirmTransaction } from "thirdweb";
+import { ClaimableERC721, ClaimableERC1155 } from "thirdweb/modules";
+import { useActiveAccount, useReadContract } from "thirdweb/react";
+import { z } from "zod";
+import { addressSchema } from "../zod-schemas";
+import { CurrencySelector } from "./CurrencySelector";
+import { ModuleCardUI, type ModuleCardUIProps } from "./module-card";
+import type { ModuleInstanceProps } from "./module-instance";
+
+export type ClaimCondition = {
+  availableSupply: bigint;
+  allowlistMerkleRoot: `0x${string}`;
+  pricePerUnit: bigint;
+  currency: string;
+  maxMintPerWallet: bigint;
+  startTimestamp: number;
+  endTimestamp: number;
+  auxData: string;
+};
+
+function ClaimableModule(props: ModuleInstanceProps) {
+  const { contract, ownerAccount } = props;
+  const account = useActiveAccount();
+
+  const primarySaleRecipientQuery = useReadContract(
+    ClaimableERC721.getSaleConfig,
+    {
+      contract: contract,
+    },
+  );
+  const claimConditionQuery = useReadContract(
+    ClaimableERC721.getClaimCondition,
+    {
+      contract: contract,
+    },
+  );
+
+  const isErc721 = props.contractInfo.name === "ClaimableERC721";
+  const isSequentialTokenIdInstalled = !!props.allModuleContractInfo.find(
+    (module) => module.name.includes("SequentialTokenId"),
+  );
+
+  const mint = useCallback(
+    async (values: MintFormValues) => {
+      if (!account) {
```

Comment:
we are completely hiding the form if the connected wallet is not the owner wallet, but we are sending the transaction using the `account` object from `useActiveAccount` - why not just use the `props.ownerAccount` here?

Reply:
Users that aren't the owner of the contract should still be allowed to mint from the claimable module (so long as they pass the claim condition requirements)

Removed the conditional so that the `MintSection` isn't restricted to only the owner account